plugins/woocommerce/src/Utilities/VersionUtil.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Automattic\WooCommerce\Utilities;
+
+use Automattic\Jetpack\Constants;
+use Automattic\WooCommerce\Proxies\LegacyProxy;
+
+/**
+ * Class VersionUtil
+ *
+ * @since x.x.x
+ *
+ * phpcs:disable Generic.Commenting.DocComment.MissingShort
+ */
+class VersionUtil {
+
+	/** @var LegacyProxy */
+	protected $proxy;
+
+	/**
+	 * Init this class instance.
+	 *
+	 * @since x.x.x
+	 * @internal
+	 *
+	 * @param LegacyProxy $proxy The legacy proxy instance.
+	 *
+	 * @return void
+	 */
+	final public function init( LegacyProxy $proxy ) {
+		$this->proxy = $proxy;
+	}
+
+	/**
+	 * Check if the current WordPress version is at least the given version.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param string $version                The version to check against.
+	 * @param bool   $use_unmodified_version Whether to use the unmodified WordPress version from
+	 *                                       the wp-includes/version.php file.
+	 *
+	 * @return bool
+	 */
+	public function wp_version_at_least( string $version, bool $use_unmodified_version = false ): bool {
+		if ( $use_unmodified_version ) {
+			$wp_version = $this->get_unmodified_wp_version();
+		} else {
+			$wp_version = $this->proxy->get_global( 'wp_version' );
+		}
+
+		return version_compare( $wp_version, $version, '>=' );
+	}
+
+	/**
+	 * Get the unmodified WordPress version from the wp-includes/version.php file.
+	 *
+	 * @since x.x.x
+	 * @return string
+	 */
+	private function get_unmodified_wp_version() {
```

Comment:
Could you add a comment that explains the difference between the global `$wp_version` and the one returned by `get_unmodified_wp_version`?

Reply:
From a previous review conversation, I got the feeling that this whole file is actually leftover from some previous work: https://github.com/woocommerce/woocommerce/pull/39701#discussion_r1376958151
Removed altogether in a429f508de99731d65ad2fc0611d235150c1d4d0